pkgs/development/python-modules/srsly/default.nix

```diff
@@ -13,13 +13,13 @@
 
 buildPythonPackage rec {
   pname = "srsly";
-  version = "2.4.0";
+  version = "2.4.1";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "e29730be53015970e4a59050e8e9f9be44d762108a617df56c9dfc981b515ab7";
+    hash = "sha256-sPKuwKMp5ufnQqCmDpmnSWjKKb5x81xcTeIh4ygXaSY=";
   };
 
   nativeBuildInputs = [ cython ];
```

Comment:
pytest no longer runs tests. Do the tests now work in the sandbox?

Also unused input pathlib.

Reply:
> pytest no longer runs tests. 

The tests are run by `setuptools-check-hook`, pytest is a module dependency for the tests.

> Do the tests now work in the sandbox?

Seems to work now, so I have enabled the tests.

> Also unused input pathlib.

Removed, also the unused dependency `pytz`.